src/org/labkey/test/BaseWebDriverTest.java

```diff
@@ -356,6 +356,8 @@ public URL getLastPageURL()
     private static long testCount;
     private static int currentTestNumber;
 
+    private static Map<String, Boolean> previousExperimentalFlags = Collections.emptyMap();
+
```

Comment:
Have `ExperimentalFeaturesHelper` keep track of previous experimental feature settings. That way, one-off calls to enable/disable experimental features can be included in the reset.
See how we set and reset `PipelineToolsHelper._originalToolsDirs`.